assets/models/system/databricks-dolly-v2-7b/spec.yaml

```diff
@@ -0,0 +1,46 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: databricks-dolly-v2-7b
+path: ./
+properties:
+  SHA: d632f0c8b75b1ae5b26b250d25bfba4e99cb7c6f
+  datasets: databricks/databricks-dolly-15k
+  evaluation-min-sku-spec: 16|0|64|128
+  evaluation-recommended-sku: Standard_NC6s_v3  
+  finetune-min-sku-spec: 12|1|64|176
+  finetune-recommended-sku: Standard_NC24s_v3 
+  finetuning-tasks: text-classification, token-classification
```

Comment:
I see that [FT job for token classification failed](https://ml.azure.com/experiments/id/dd332e12-ceee-4be5-9394-b9e3785916d4/runs/magenta_planet_ldg99xhrh3?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-kartik2&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#). Is there a reason to include it as well despite failure?

Reply:
Token classification Finetune and Model Prediction is passing. It is failing in Compute Metrics Process due to a known issue. Should I keep it or remove it?

Reply:
@skasturi  I removed the token-classification as of now.